From 7269d27dc800a3913d75b7f2afd9f5f63a9d5e8e Mon Sep 17 00:00:00 2001
From: DiaDust <diadust0514@gmail.com>
Date: Sun, 27 Oct 2013 02:01:17 +0900
Subject: [PATCH] error: Remove drivers/gpu/arm directory on driver/Makefile
 driver/video/Kconfig extends this commit
 5cc1d6208b1e4cf0e3f47d8da1ba24e4b8ff4c04

---
 drivers/gpu/Makefile  | 2 +-
 drivers/video/Kconfig | 2 --
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/gpu/Makefile b/drivers/gpu/Makefile
index c859db5..59ae0d6 100644
--- a/drivers/gpu/Makefile
+++ b/drivers/gpu/Makefile
@@ -1,2 +1,2 @@
-obj-y			+= drm/ vga/ stub/ ion/ arm/
+obj-y			+= drm/ vga/ stub/ ion/
 obj-y			+= pvr/
diff --git a/drivers/video/Kconfig b/drivers/video/Kconfig
index af39149..1088bf9 100644
--- a/drivers/video/Kconfig
+++ b/drivers/video/Kconfig
@@ -25,8 +25,6 @@ source "drivers/gpu/stub/Kconfig"
 
 source "drivers/gpu/ion/Kconfig"
 
-source "drivers/gpu/arm/Kconfig"
-
 source "drivers/gpu/pvr/Kconfig"
 
 config VGASTATE
-- 
1.8.4


